module_intros/locality.Filter.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>In the equation, should it be 2pi = (1 - r_{I, j} / R) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>good catch, I fixed it now</p>